docs/developers_guide/ocean/test_groups/ziso.rst

```diff
@@ -58,31 +58,31 @@ and the time steps for each resolution:
 .. code-block:: python
 
     res_params = {'20km': {'cores': 20,
-                           'min_cores': 2,
+                           'min_tasks': 2,
                            'cores_with_particles': 32,
                            'min_cores_with_particles': 12,
```

Comment:
Here and below, the changes are quite a bit more substantial:
https://github.com/MPAS-Dev/compass/blob/fef3aded3efb7dd44a483d0076fa8fb566184976/compass/ocean/tests/ziso/forward.py#L57-L89